### PR TITLE
2.5 Privacy: talk about mitigating harm

### PR DESCRIPTION
**Pull request by rhiaro**
… using text from @simoneonofri and @mnot in #112

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -225,8 +225,9 @@ <h3 id="privacy" data-export="" data-dfn-type="dfn">
 their financial transactions
 and how they live their lives.
 We will start by creating web technologies
-that create as few risks as possible,
-and will make sure people understand
+that create as few potential harms as possible,
+and mitigate the harms that we cannot avoid.
```
Comment (rhiaro) [inline index.html]:
```suggestion
that create as few potential threats as possible,
and mitigate the threats that we cannot avoid.
```

File: index.html [hunk 2]
```diff
@@ -225,8 +225,9 @@ <h3 id="privacy" data-export="" data-dfn-type="dfn">
 their financial transactions
 and how they live their lives.
 We will start by creating web technologies
-that create as few risks as possible,
-and will make sure people understand
+that create as few potential threats as possible,
```
Comment (rhiaro) [inline index.html]:
```suggestion
that create as few potential threats to web users as possible,
```

**Review by mnot — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -225,8 +225,9 @@ <h3 id="privacy" data-export="" data-dfn-type="dfn">
 their financial transactions
 and how they live their lives.
 We will start by creating web technologies
-that create as few risks as possible,
-and will make sure people understand
+that create as few potential threats to web users as possible,
+and mitigate the threats that we cannot avoid.
+We will will make sure people understand
```
Comment (martinthomson) [inline index.html]:
```suggestion
We will make sure people understand
```

(Separate thought: who owns this responsibility?  Can we really say that "we" - for some relevant value of "we" - can genuinely make sure?)

**Review by torgo — APPROVED** [review]
lgtm!

**Pull request merged**